source/index.txt

```diff
@@ -60,9 +60,21 @@ Available |compass-short| Editions
        write and delete capabilities removed.
 
    * - :guilabel:`Compass Isolated`
-     - Does not initiate any network requests
-       except to the MongoDB server to which |compass-short| connects.
-       This edition is designed for highly secure environments.
+     - Does not initiate any network requests except to the MongoDB
+       server to which |compass-short| connects. This edition is
+       designed for highly secure environments.
+       
+       Isolated edition can perform DNS alias lookups and leverage A
+       Records.
```

Comment:
```suggestion
       Isolated edition can perform DNS queries (e.g. lookup A or SRV
       Records).
```
I'd stay vague here, with a couple of examples in parentheses. Any DNS query will work, no matter if it's to lookup an A record, or a CNAME or if the cluster users SRV records.